tools/find_untranslated_strings.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
no. But the line has no effect in this script, since it doesn't use any utf8 chars.